integration/overview.mdx

```diff
@@ -3,33 +3,67 @@ title: Overview
+Flipt comes equipped with a fully functional REST API. The Flipt UI is completely backed by this same API. This means that anything that can be done in the Flipt UI can also be done via the REST API.
 
-The Flipt REST API can also be used with any language that can make HTTP requests. This means that you don't need to use one of the above GRPC clients to integrate your application with Flipt.
+The Flipt REST API can also be used with any language that can make HTTP requests. This means that you don't need have to use one of our official clients to integrate your application with Flipt.
 
 The latest version of the REST API is fully documented using the [OpenAPI v3 specification](https://github.com/flipt-io/flipt-openapi) as well as the above [API Reference](/reference/overview).
 
-See all official REST clients as well as how to generate your own in the the [REST SDK](/integration/rest) section.
+See all official REST SDKs as well as how to generate your own in the the [REST SDK](/integration/server/rest) section.
 
-## GRPC API
+### GRPC API
 
 Since Flipt is a [GRPC](https://grpc.io/) enabled application, you can connect to it using the GRPC protocol. This means that you can use any language that has a GRPC client implementation to integrate with Flipt.
 
+<Note>GRPC requires HTTP/2 in your environment.</Note>
+
 An example [Go application](https://github.com/flipt-io/flipt/tree/main/examples/basic) is available, showing how you would
 integrate with Flipt using the Go GRPC client.
 
-See all official GRPC clients as well as how to generate your own in the the [GRPC SDK](/integration/grpc) section.
+See all official GRPC SDKs as well as how to generate your own in the the [GRPC SDK](/integration/server/grpc) section.
+
+## Client-Side Evaluation
+
+Client-side evaluation is another way Flipt supports evaluating feature flags. This is where your application has a local copy of the feature flag rules and evaluates the feature flag locally.
+
+<Warning>
+  Our client-side SDKs are currently in **beta**.
+
+We're actively working on improving them and adding more languages. If you have any feedback, feature or language requests, please let us know by opening an issue in our [GitHub repository](https://github.com/flipt-io/flipt-client-sdks).
+
+</Warning>
+
+Client-side evaluation is much more performant than server-side evaluation, but it comes with some tradeoffs. The main tradeoff is that you need to keep your feature flag rules in sync with Flipt. This means that you will need to periodically fetch the feature flag rules from Flipt and update your local copy. Our client-side SDKs provide a way to do this automatically.
+
+Reasons for using client side evaluation include:
+
+- Your application is running in an environment where it cannot make requests to Flipt for each feature flag evaluation (for example a mobile application)
```

Comment:
Im not sure this makes sense. Since the client still needs to talk to Flipt. It just does so asynchronously from evaluations.

Reply:
Yeah perhaps it could be rewritten but it is true, it doesn't make a request for each evaluation. I wanted to suggest that its useful for environments where the network is not always %100 reliable

Reply:
i guess that is covered in the point below though. will drop